docs/components/motor/roboclaw.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Configure a roboclaw motor"
+linkTitle: "roboclaw"
+weight: 22
+type: "docs"
+description: "Configure a brushed DC motor driven by a roboclaw motor controller."
+images: ["/components/img/components/motor.svg"]
+# SMEs: Olivia, Rand
+---
+
+The `roboclaw` model of the motor component supports [standard brushed DC motors](https://en.wikipedia.org/wiki/DC_motor) driven by [Basicmicro's](https://www.basicmicro.com/) [RoboClaw](https://www.basicmicro.com/RoboClaw-2x30A-Motor-Controller_p_9.html) motor controller.
+
+{{< alert title="Note" color="note" >}}
+
+You must set up your RoboClaw before configuring it.
```

Comment:
[nit] Is this order of operations necessary because you wouldn't know the motor channel or something until after you set it up?